test/internalTest.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-env mocha */
+
+const versions = require('../src/version')
+
+for (const supportedVersion of versions.testedVersions) {
+  describe(`prismarineEntity ${supportedVersion}`, function () {
+    const PrismarineEntity = require('../')(supportedVersion)
```

Comment:
This thing does not support parallel testing, please look at how other PrismarineJS projects are being tested.